scala/private/phases/phase_test_environment.bzl

```diff
@@ -1,14 +1,28 @@
 # Adds testing.TestEnvironment provider if "env" attr is specified
 # https://bazel.build/rules/lib/testing#TestEnvironment
 
+load("@bazel_skylib//lib:versions.bzl", "versions")
+
+
 def phase_test_environment(ctx, p):
     test_env = ctx.attr.env
+    inherited_environment = ctx.attr.inherited_environment
 
-    if test_env:
+    if inherited_environment:
```

Comment:
Note for reviewer: I removed the original if as `test_env` has default valued of `{}` so it is always set. Unless I am wrong, OFC :)

Reply:
In Starlark, an empty dictionary evaluates to `False`: "A dictionary used in a Boolean context is considered true if it is non-empty."

Reply:
Ahhh right! Thanks :) This makes a lot more sense now - I assumed this is TypeScript, so it's true if it's not null...